### DISCUSSION TITLE
Histogram counter drops to 0 and then returns to the last value during vector pod restart

### DISCUSSION THREAD

**Discussion by ShukhratE** [vectordotdev/vector#15806] [Q&A]
Hi,
I am using prometheus_remote_write to send metrics.
During vector pods restart I see strange behavior when the histogram_counter metric is reset to 0 and then shows the last known state.

![image](https://user-images.githubusercontent.com/58506422/210589190-1564ce70-9e10-4265-a944-82fedc6af51f.png)

Any idea what can cause such an outcome? 

Config of remote_write:
```
    [sinks.victoria_local]
      type = "prometheus_remote_write"
      inputs = ["vector_metrics_source"]
      endpoint = "http://10.187.0.47:8428/api/v1/write"
      buffer.type = "memory"
      buffer.max_events = 60000
      buckets = [ 0.005, 0.1, 0.5, 1, 3, 20 ]
      healthcheck.enabled = false
      default_namespace = "service" # required
```

**Comment by jszwedko** [comment]
This was discussed in Discord: https://discord.com/channels/742820443487993987/1060253332586893433/1060253332586893433
